upoi/map/models.py

```diff
@@ -1,3 +1,70 @@
 from django.db import models
 
-# Create your models here.
+
+class User(models.Model):
+    user = models.CharField(max_length=20)
+    name = models.CharField(max_length=20)
+    lastname = models.CharField(max_length=20)
+    email = models.EmailField(max_length=50)
+    layer = models.ManyToManyField('Layer',
+        related_name='users_layers'
+    )
+    favorite_layers = models.ManyToManyField(
+        'Layer',
+        related_name='users_favorite_layers',
+    )
+
+
+class TypeOfPoint(models.Model):
+    pass
+
+
+class Background(models.Model):
+    pass
+
+
+class Link(models.Model):
+    pass
+
+
+class Point(models.Model):
+    latitude = models.DecimalField(
+        decimal_places=2,
+        max_digits=10
+    )
+    longitude = models.DecimalField(
```

Comment:
Instead of using a lat and lng, we should use the Point model from geo: https://docs.djangoproject.com/en/1.10/ref/contrib/gis/model-api/#pointfield